packages/protocol/package.json

```diff
@@ -6,61 +6,61 @@
-    "lint:sol": "solhint --version && solhint './contracts/**/*.sol' && solhint './contracts-0.8/**/*.sol'",
-    "lint": "yarn run lint:ts && yarn run lint:sol",
+    "anvil-devchain:e2e-tests": "./scripts/foundry/run_e2e_tests_in_anvil.sh",
+    "anvil-devchain:integration-tests": "./scripts/foundry/run_integration_tests_in_anvil.sh",
+    "anvil-devchain:start-L1": "./scripts/foundry/create_and_migrate_anvil_devchain.sh",
+    "anvil-devchain:start-L2": "./scripts/foundry/create_and_migrate_anvil_l2_devchain.sh",
+    "anvil-devchain:status": "if nc -z localhost 8546; then echo 'Devchain is serving at http://localhost:8546'; else echo 'Devchain is not running.'; fi",
+    "anvil-devchain:stop": "./scripts/foundry/stop_anvil.sh",
+    "artifacts:download": "./scripts/bash/download_artifacts.sh",
+    "artifacts:upload": "./scripts/bash/upload_artifacts.sh",
+    "prebuild": "rm -rf ./build",
+    "build": "yarn build:sol && yarn build:ts",
+    "build:sol": "yarn submodules:pull && mkdir -p migrations && yarn ts-node --preferTsExts ./scripts/build.ts --solidity ${BUILD_DIR:-./build}",
+    "build:ts": "rm -f migrations/*.js* && yarn ts-node --preferTsExts ./scripts/build.ts --truffleTypes ./types/typechain && tsc -b && mv migrations_ts/*.js* migrations",
+    "check-opcodes": "yarn ts-node scripts/check-opcodes.ts",
+    "check-versions": "./scripts/bash/check-versions.sh",
+    "ci:test-make-release": "./scripts/bash/release-on-devchain.sh",
     "clean": "rm -rf ./types/typechain && rm -rf build/* && rm -rf .0x-artifacts/* && rm -rf migrations/*.js* && rm -rf migrations_ts/*.js* && rm -rf test/**/*.js* && rm -f lib/*.js* && rm -f lib/**/*.js* && rm -f scripts/*.js* && yarn clean:foundry",
     "clean:foundry": "forge clean && rm -rf cache out",
-    "test": "rm test/**/*.js ; node runTests.js",
-    "test:scripts": "yarn ts-node scripts/run-scripts-tests.ts --testPathPattern=scripts/",
-    "quicktest": "./scripts/bash/quicktest.sh",
-    "test:coverage": "yarn run test --coverage",
-    "ci:test-make-release": "./scripts/bash/release-on-devchain.sh",
-    "test:release-snapshots": "./scripts/bash/release-snapshots.sh",
-    "test:generate-old-devchain-and-build": "./scripts/bash/generate-old-devchain-and-build.sh",
-    "build:ts": "rm -f migrations/*.js* && yarn ts-node --preferTsExts ./scripts/build.ts --truffleTypes ./types/typechain && tsc -b && mv migrations_ts/*.js* migrations",
-    "gas": "yarn run test --gas",
-    "pull-submodules": "git submodule update --init --recursive",
-    "delete-submodules": "rm -rf $(git submodule | awk '{ print $2 }')",
-    "build:sol": "yarn pull-submodules && mkdir -p migrations && yarn ts-node --preferTsExts ./scripts/build.ts --solidity ${BUILD_DIR:-./build}",
-    "build": "yarn build:sol && yarn build:ts",
-    "prebuild": "rm -rf ./build",
+    "console": "./scripts/bash/console.sh",
     "determine-release-version": "yarn --silent ts-node --preferTsExts ./scripts/determine-release-version.ts",
```

Comment:
Wondering if this is used and what are exact release steps.
Maybe could be bundled all together under `release:...`

Reply:
This wasn't a necessary step of the release process, just a helpful dev tool back when Truffle was used. This started `truffle console`, which was just a Node REPL with web3/Truffle in the environment. I'd keep it as is, to be removed when Truffle is gone.

Reply:
Highlighted part was `determine-release-version` and I believe you gave details (still helpful) on `console` 😉

Reply:
Leaving as a tool for release under `release:determine-release-version`, resolving 😉